cmd/main.go

```diff
@@ -55,6 +56,7 @@ func init() {
 	utilruntime.Must(sourcev1.AddToScheme(scheme))
 	utilruntime.Must(hcv2.AddToScheme(scheme))
 	utilruntime.Must(sveltosv1beta1.AddToScheme(scheme))
+	utilruntime.Must(capo.AddToScheme(scheme))
```

Comment:
I don't see any related changes to the controller which are using openstack types, so it looks completely out of scope of this PR.
Remove.

Reply:
Same as above.